about.css

```diff
@@ -0,0 +1,134 @@
+@font-face {
+  font-family: 'Lato';
+  src: url('./fonts/Lato-Regular.ttf') format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', serif;
+}
+
+.ab-headline {
+  background: url('./imgs/container-bg.jpeg');
+}
+
+.ab-headline-div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.ab-headline-div h2 {
+  color: #f34e3e;
+  font-size: 20.6px;
+  padding-top: 17px;
+}
+
+.ab-headline-div h1 {
+  color: #f7423f;
+  font-weight: bold;
+  font-size: 32.6px;
+  padding-bottom: 50px;
+  padding-top: 50px;
+}
+
+.ab-headline-div p {
+  background: #fff;
+}
```

Comment:
- [ ] Please use the same border in this section to match the given design and increase the quality of the look.

yours|given
---|---
![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/91486791/4e05cb3b-9f62-43cc-bc4e-8d53ae50d937)|![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/91486791/996528b0-355d-414c-a739-366a9635da2d)